ABloom/ABloom/Presentation/CoreViews/QnA/View/AnswerCheckView.swift

```diff
@@ -48,6 +47,9 @@ struct AnswerCheckView: View {
       .padding(.horizontal, 20)
       .background(backWall())
     }
+    .onAppear {
+      answerCheckVM.getAnswers()
+    }
```

Comment:
```suggestion
    .task {
      answerCheckVM.getAnswers()
    }
```
뷰와 동일한 수명을 가질 수 있도록 하기 위해! .task로의 수정을 제안합니다!